frontend/src/app/[locale]/search/page.tsx

```diff
@@ -60,47 +64,66 @@ function Search({ searchParams, params }: SearchPageProps) {
   const searchResultsPromise = searchForOpportunities(convertedSearchParams);
 
   return (
-    <>
+    <QueryProvider>
       <SearchAnalytics
         params={resolvedSearchParams}
         newRelicEnabled={environment.NEW_RELIC_ENABLED === "true"}
       />
-      <QueryProvider>
-        <div className="grid-container">
-          <div className="search-bar">
-            <SearchBar queryTermFromParent={query} />
-          </div>
-          <div className="grid-row grid-gap">
-            <div className="tablet:grid-col-4">
-              <ContentDisplayToggle
-                showCallToAction={t("filterDisplayToggle.showFilters")}
-                hideCallToAction={t("filterDisplayToggle.hideFilters")}
-                breakpoint={Breakpoints.TABLET}
-                type="centered"
-              >
-                <SaveSearchPanel />
-                <SearchFilters
-                  opportunityStatus={status}
-                  eligibility={eligibility}
-                  category={category}
-                  fundingInstrument={fundingInstrument}
-                  agency={agency}
-                  topLevelAgency={topLevelAgency}
-                  searchResultsPromise={searchResultsPromise}
-                />
-              </ContentDisplayToggle>
-            </div>
-            <div className="tablet:grid-col-8">
-              <SearchResults
-                searchParams={convertedSearchParams}
-                loadingMessage={t("loading")}
+      <BetaAlert
```

Comment:
beta alert, breadcrumbs and CTA moved out of layout in order to support putting them behind the background color on the new page, so they need to be added in here as well